website/docs/r/backup_compliance_policy.html.markdown

```diff
@@ -17,6 +17,15 @@ When enabled, the Backup Compliance Policy will be applied as the minimum backup
 
 -> **NOTE:** With Backup Compliance Policy enabled, cluster backups are retained after a cluster is deleted and backups can be used normally until retention expiration. When the Backup Compliance Policy is not enabled, Atlas deletes the cluster's associated backup snapshots when a cluster is terminated. By default, a Backup Compliance Policy is not enabled. For more details see [Back Up, Restore, and Archive Data](https://www.mongodb.com/docs/atlas/backup-restore-cluster/). 
 
+-> **NOTE:** If you encounter errors when deleting an Atlas Cluster which has an associated `mongodbatlas_cloud_backup_schedule` resource, we suggest the following: 
+* `mongodbatlas_cloud_backup_schedule` fails when you delete an Atlas cluster because an existing backup compliancy policy is enabled, which is expected.
+We first suggest disabling `mongodbatlas_backup_compliancy_policy` resource which requires contacting MongoDB Support and completing an extensive verification process.
+* Second, if the first suggestion is not possible, we suggest changing the order in which terraform deletes the resource. This might not always be possible since the `mongodbatlas_cloud_backup_schedule` resource is referencing attributes from the `mongodbatlas_advanced_cluster`, or `mongodbatlas_cluster resource`, and therefore has to delete the backup schedule before deleting the cluster resource. This dependency must be defined as stated in the `mongodbatlas_cloud_backup_schedule` in the [documentation](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule).
```

Comment:
nit: I am not sure about suggesting to disable `backup_compliancy_policy` as the first option, as this is likely the most time consuming and not ideal. Was the intention here to provide more context on the limitations?

Reply:
Believe that is up to customer to determine. Agree likely most time consuming, but by contacting MongoDB Support and user has completed extensive verification process, they is high probability that our Support Engineer will be able to help user to delete resource directly. Also serves has helpful reminder of why this limitation on deleting `mongodbatlas_cloud_backup_schedule` exists on the first place for users with `mongodbatlas_backup_compliance_policy` enabled